docs/product-assembly_pam-on-openshift/src/main/asciidoc/ba-openshift-overview-con.adoc

```diff
@@ -0,0 +1,35 @@
+[id='ba-openshift-overview-con']
+= Overview
+If you have an OpenShift environment, you can deploy {PRODUCT} into this environment.
+
+In this solution, components of {PRODUCT} are deployed as separate OpenShift pods. You can scale each of the pods up and down individually, providing as few or as many containers as necessary for a particular component. You can use standard OpenShift methods to manage the pods and balance the load.
+
+The following key components are available as pods running in OpenShift:
+
+* {KIE_SERVER}, also known as _Execution Server_ or _KIE Server_, is the infrastructure element that runs a process or a group of processes. All process logic runs on execution servers.
++
+A database backend is required for an execution server. You can provide a database server in another OpenShift pod or configure an execution server on OpenShift to use any other database server.
```

Comment:
If a database is not provided then is used in memory h2 DB (Database in this case is not persistent and Kie Server scaling will not work)

Reply:
@jakubschwan apart from "no scaling" what is the danger of a non-persistent DB? Do I understand correctly that a production process can end up giving the wrong result because the pod was restarted between calls?